pyperf/pyperf_run

```diff
@@ -272,6 +272,9 @@ if [ $to_pbench -eq 0 ]; then
 	# Install pip/pip3
 	#
 	wget https://bootstrap.pypa.io/get-pip.py
```

Comment:
While -ENOTTHISPR, this isn't the way we expect people to get pip on RHEL/Fedora I believe, it should be `dnf install python3-pip`.